oteapi/models/mappingconfig.py

```diff
@@ -15,15 +15,18 @@ class MappingConfig(GenericConfig):
     mappingType: str = Field(
         ...,
         description="Type of registered mapping strategy.",
+        IRI="http://purl.org/dc/terms/type",  # type: ignore
     )
     prefixes: Optional[Dict[str, str]] = Field(
         None,
         description=(
             "Dictionary of shortnames that expands to an IRI given as local "
             "value/IRI-expansion-pairs."
         ),
+        IRI="http://www.w3.org/2004/02/skos/core#notation",  # type: ignore
```

Comment:
If using `skos:notation` for prefixes, then we need a to define a custom datatype, like `oteio:prefixType`, such that we can serialise a prefix as:

    :mappingFilter skos:notation "rdfs: <http://www.w3.org/2000/01/rdf-schema#>"^^oteio:prefixType .

However, I think it would be simpler to define our own `oteio:prefix`, such that we can express the above as:

    :mappingFilter oteio:prefix "rdfs: <http://www.w3.org/2000/01/rdf-schema#>" .